python/taichi/main.py

```diff
@@ -173,6 +193,10 @@ def make_argument_parser():
                         '--gui',
                         action='store_true',
                         help='Display benchmark regression result in GUI')
+    parser.add_argument('-T',
+                        '--tprt',
+                        action='store_true',
+                        help='Run TPRT benchmark instead of SPRT')
```

Comment:
We may want the full name in the help as users probably don't know what `TPRT` and `SPRT` mean.